src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -201,7 +198,7 @@ const ChatItemMyTextV2 = ({
   }, []);
 
   const renderAvatar = React.useCallback(() => {
-    if (isContinuous) return <View style={styles.avatar} />;
+    if (isContinuous) return <View style={[styles.avatar, styles.ml8]} />;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The renderAvatar function now includes the styles.ml8 class when the isContinuous condition is true. This likely affects the margin around the avatar in certain scenarios. Ensure this change doesn't disrupt the layout.
```diff
- if (isContinuous) return <View style={[styles.avatar, styles.ml8]} />;
+ if (isContinuous) return <View style={styles.avatar} />;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->